test/test.js

```diff
@@ -75,6 +75,14 @@ describe('status', function () {
     })
   })
 
+  describe('.STATUS_CODES', function () {
+    it('should include status code map from Node.js', function () {
+      Object.keys(http.STATUS_CODES).forEach(function forEachCode (code) {
+        assert.notEqual(status.STATUS_CODES[code], -1, 'contains ' + code)
```

Comment:
I don't understand this test. When would the value equal `-1`? Were you trying to use `.indexOf` or something?

Reply:
That's actually a great question. I can improve this test to be better - give me a minute.
This was copied from https://github.com/jshttp/statuses/blob/master/test/test.js#L81.